disposable_email_blocklist.conf

```diff
@@ -1914,6 +1915,7 @@ mirai.re
 misterpinball.de
 miucce.com
 mji.ro
+mjj.edu.ge
```

Comment:
How can I generate a disposable email from this domain?

Reply:
https://mail.mjj.edu.ge/